### PR TITLE
Ch qubitekk update

### PR DESCRIPTION
**Pull request by CatherineH**
This is a clean up of the original qubitekk code, and an update to the latest version of the firmware. I have not ensured 100% test coverage.

### CODE REVIEW

**Comment by CatherineH** [comment]
Okay, the qubitekk cc1 is at 100% coverage. This is ready to merge.

File: instruments/instruments/qubitekk/cc1.py [hunk 1]
```diff
@@ -52,8 +65,11 @@ def __init__(self, filelike):
         self.terminator = "\n"
         self.end_terminator = "\n"
         self.channel_count = 3
+        self.channel1 = self.Channel(self, 02)
```
Comment (scasagrande-d2l) [inline instruments/instruments/qubitekk/cc1.py]: You already have `CC1.channel` defined as a property using `ProxyList`, just remove these.

File: instruments/instruments/qubitekk/cc1.py [hunk 2]
```diff
@@ -52,8 +65,11 @@ def __init__(self, filelike):
...
+        self.channel2 = self.Channel(self, 1)
+        self.channelc = self.Channel(self, 2)
```
Comment (scasagrande-d2l) [inline instruments/instruments/qubitekk/cc1.py]: What is `.channelc` ?

File: instruments/instruments/qubitekk/cc1.py [hunk 3]
```diff
@@ -63,7 +76,7 @@ class Channel(object):
         __CHANNEL_NAMES = {
             1: 'C1',
             2: 'C2',
-            3: 'C0'
+            3: 'CO'
```
Comment (scasagrande) [inline instruments/instruments/qubitekk/cc1.py]: Why does `CC1.channel[2]` end up mapping to `C0`?

Comment (scasagrande) [inline instruments/instruments/qubitekk/cc1.py]: Oh, that's `CO` not `C0`

**Comment by CatherineH** [comment]
I'm closing this PR because I've been adding my changes to the branch on the main fork.

**Pull request closed without merging**